weddingApi/src/main/java/com/weddingapi/services/DeviceTokenResource.java

```diff
@@ -49,4 +52,14 @@ public Response add(final String jsonString) {
             session.close();
         }
     }
+
+    /**
+     * To retreive device tokens from wedding object -
+     *
+     session.createCriteria(DeviceToken.class).add(Restrictions.eq(
+     "Wedding",wedding
+     )).list()
+
+     */
+    //TODO try removing wedding.getdevicetokens() same from event.getWeddingEvents
```

Comment:
Currently the wedding object does not store device tokens associated to it. To get the device token you need to do something like -
`session.createCriteria(DeviceToken.class).add(Restrictions.eq(
     "Wedding",wedding
     )).list()`

I am unable to figure out why this is happening. Kept it as TODO as did not feel it to be a big hurdle now